### PR TITLE
ui: enable order warning acknowledgement

### LINKED ISSUE #1985: ui: user should be able to acknowledge order disclaimer
image

Should be able to hide that message somehow.

### PR DESCRIPTION
**Pull request by buck54321**
Resolves #1985
Allows the user to hide the warning message on the order verification form. This will hide the message and display instead an option to show the warning again. 

image
---
image

`Window.localStorage` is used to remember the user's choice.

### CODE REVIEW

File: client/webserver/site/src/css/main_dark.scss
Comment (buck54321) [inline client/webserver/site/src/css/main_dark.scss:21]: Unrelated to the goal, but I felt like this needed a tweak.

**Comment by norwnd** [comment]
Seems to be working nicely, and LGTM overall.

**Review by martonp — APPROVED** [review]

**Review by chappjc — APPROVED** [review]

File: client/webserver/site/src/js/state.ts
Comment (chappjc) [inline client/webserver/site/src/js/state.ts:10]: I'm sure you see @norwnd, but please note this for https://github.com/decred/dcrdex/pull/1975

Comment (norwnd) [reply client/webserver/site/src/js/state.ts:10]: yep (and appreciate you pointing this out! better safe...), 

I feel like those commits in https://github.com/decred/dcrdex/pull/1975/commits need to be squashed before I rebase (and we merge into master),

but I believe I should get final approval in https://github.com/decred/dcrdex/pull/1975 first, so that reviewers can focus on simple diffs for latests commits, just to clarify I've got your review-process approach right @chappjc ?

**Pull request merged**
